src/courtroom.cpp

```diff
@@ -3836,7 +3836,7 @@ void Courtroom::set_self_offset(QString p_list, QString p_effect) {
       self_offset_v = 0;
     else
       self_offset_v = self_offsets[1].toInt();
-    ui_vp_player_char->move(ui_viewport->width() * self_offset / 100, ui_viewport->height() * self_offset_v / 100);
+    ui_vp_player_char->move_and_center(ui_viewport->width() * self_offset / 100, ui_viewport->height() * self_offset_v / 100);
```

Comment:
this seems like it breaks offsets, have you tested this?

Reply:
Yeah, the offsets work with this, tested on alpha-2.10 branch actions build